src/main/scala/com/amazon/deequ/comparison/DataSynchronization.scala

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.comparison
+
+import org.apache.spark.sql.{Column, DataFrame}
+import org.apache.spark.sql.functions.col
+
+object DataSynchronization {
+
+  /**
+   * Compare two DataFrames 1 to 1 with specific columns inputted by the customer.
```

Comment:
can you add an example for how this will be used with the params below?

Reply:
Example added.